### PR TITLE
1657269: Do not use /var/run, but use /run; ENT-1086

### PR DESCRIPTION
**Pull request by jirihnidek**
* BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1657269
* The directory `/var/run` is marked as obsoleted and it is
  also hard link to `/run`
* Chnaged all occurance of `/var/run` to `/run`
* Prepared the code for similar change in virt-who

### CODE REVIEW

**Comment by jirihnidek** [comment]
More details in this article: https://access.redhat.com/solutions/4154291

File: Makefile [hunk 1]
```diff
@@ -18,6 +18,7 @@ DESTDIR ?= /
 PREFIX ?= /usr/local
 SYSCONF ?= etc
 INSTALL_DIR = $(PREFIX)/share
+# RUN_DIR ?= /run
```
Comment (wottop) [inline Makefile:21]: Should this remain commented?

Comment (jirihnidek) [reply Makefile]: Good point. No, it should not be commented. When somebody tries to use make without arguments, then some default value should be defined in `Makefile`.

**Comment by wottop** [comment]
ack

**Pull request merged**